sale_exception/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,5 @@
+To configure this module you need to:
+
+#. Go to *Settings > User & Companies > Users*.
+#. Edit your user.
+#. Check "A warning can be set on a product or a customer (Sale)" group.
```

Comment:
You could maybe include here that the user should install `sale_management` to use the module, although it is not strictly a dependency...